packages/frontend/src/contexts/game/actions/index.test.js

```diff
@@ -13,11 +13,11 @@ describe('GameContext actions', () => {
   });
 
   it(`returns an action object with type ${types.FLIP_TILE} and a payload containing a tile object`, () => {
-    const tile = mockPhotos[0];
+    const tile = {tile: mockPhotos[0]}
```

Comment:
`mockPhotos[0]` is a photo, not a tile. This would have to be `const tile = createTilesFromPhotos(mockPhotos, { shuffle: false })[0]` Also, check out the value of `tile` because it looks like it's adding one extra depth to the object `{ tile: { tile: { ... } } }` vs `{ tile: { ... } }`.

Reply:
I noticed a few places where the `createTilesFromPhotos` has a boolean for the second argument instead of an options object with `{ shuffle: boolean }`